managed-ledger/src/main/proto/MLDataFormats.proto

```diff
@@ -78,6 +81,11 @@ message MessageRange {
     required NestedPositionInfo upperEndpoint = 2;
 }
 
+message BatchDeletedIndexInfo {
```

Comment:
The name is not very clear. I think it is keeping the deletion index for a batched entry. So I would suggest naming it "BatchedEntryDeletionIndexInfo"